WMIAdventure/frontend/src/js/pages/LoginPage/LoginPage.js

```diff
@@ -12,10 +12,10 @@ class LoginPage extends React.Component {
                 <Helmet>
                     <title>Login Page</title>
                 </Helmet>
-                <StyledWrapper>
+                <MainContainer>
```

Comment:
Tu pewnie ta zmiana powoduje ten błąd na loginie

Reply:
No Webstorm mi jakoś dziwnie to podmienił jak zmieniałem nazwę